src/appleseed.shaders/include/appleseed/fractal/as_fractal_helpers.h

```diff
@@ -34,25 +34,33 @@
 #define AS_FRACTAL_HELPERS_H
 
 #include "appleseed/fractal/as_noise_helpers.h"
-#include "appleseed/math/as_math_helpers.h"
 
 float fBm(
     point surface_point,
+    float initial_time,
     float filter_width,
     float amplitude,
     int octaves,
     float lacunarity,
     float gain)
 {
-    point pp = surface_point;
-    float amp = amplitude, fw = filter_width, sum = 0.0;
+    point xyz = surface_point;
+    float amp = amplitude, filter_size = filter_width;
+    float current_time = initial_time, sum = 0.0;
 
     for (int i = 0; i < octaves; ++i)
     {
-        sum += amp * filtered_snoise(pp, fw);
+        if (!amp)
```

Comment:
Ah ok, I hadn't paid enough attention.

May I suggest `amp == 0.0` though? I think this would be clearer...